Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apt-get update -y \
 
 # Install Hadoop
 RUN cd /tmp \
-    && curl -Lo - http://ftp.unicamp.br/pub/apache/hadoop/common/hadoop-3.0.0/hadoop-3.0.0.tar.gz | tar -xz \
+    && curl -Lo - https://archive.apache.org/dist/hadoop/core/hadoop-3.0.0/hadoop-3.0.0.tar.gz | tar -xz \
```

Comment:
Hadoop 3.0.0 isn't available from http://ftp.unicamp.br/pub/apache/hadoop/common/hadoop-3.0.0/hadoop-3.0.0.tar.gz any more. 

Hadoop 3.0.1 is available from http://ftp.unicamp.br/pub/apache/hadoop/common/hadoop-3.0.1/hadoop-3.0.1.tar.gz 

But I thought it might be better to use an https source for some level of validation.

Validation of checksum/signature on downloads would be another option and allow use of a wider variety of mirrors. But that's a bigger change as checksuming/signature checking differs by source and client platform.

Reply:
I think this was the reason our builds were failing here. This is a :+1: fix for me!